plugins/modules/mkfilt.py

```diff
@@ -637,8 +638,8 @@ def main():
         type='dict',
         options=dict(
             default=dict(type='str', choices=['permit', 'deny']),
-            log=dict(type='bool'),
-            force=dict(type='bool'),
+            log=dict(type='bool', default=False),
```

Comment:
@robinvy Please, backout that change.
log has to be None by default, not False.
The idea is that if log is not set, it is not changed. Setting it to False by default would always disable logging if the user does not specify it, which is not what we want.

See this test in the code:
`if params[version]['log'] is not None:`

The documentation should be changed instead of the code.

Reply:
Fixed in [f9f68f0e7fd1db39b55fa3b636647d9f2bb14dcd](f9f68f0e7fd1db39b55fa3b636647d9f2bb14dcd)